modules/end-user-guide/pages/microsoft-visual-studio-code-open-source-ide.adoc

```diff
@@ -27,6 +27,11 @@ You can automate installation of Microsoft Visual Studio Code extensions from th
 
 ====
 
+[TIP]
+====
+Configure IDE preferences on a per-workspace basis by invoking the link:https://code.visualstudio.com/api/ux-guidelines/command-palette[Command Palette] and selecting *Preferences: Open Workspace Settings*.
```

Comment:
Are we allowed to link to the upstream M$ product ? Given we build from the OSS repo, not repackage/redist their stuff (because of the super restrictive TOU), just wondering if this link is allowed.

Reply:
That was something I was wondering too; there are links above this too. @deerskindoll @themr0c have you any ideas?

Reply:
@nickboldt looks like we're set based on https://redhat-documentation.github.io/supplementary-style-guide/#external-links